sda/tree_test.go

```diff
@@ -43,7 +42,7 @@ func TestTreeId(t *testing.T) {
 
 // Test if topology correctly handles the "virtual" connections in the topology
 func TestTreeConnectedTo(t *testing.T) {
-	names := genLocalhostPeerNames(3, 0)
+	names := genLocalhostPeerNames(3, 2000)
```

Comment:
If we can get it to work with `:0`, that would be great. Else parallel tests will fail.